src/SourceLocator/Ast/Strategy/NodeToReflection.php

```diff
@@ -47,6 +49,17 @@ public function __invoke(
             );
         }
 
+        if ($node instanceof Node\Stmt\Const_) {
+            return ReflectionConstant::createFromNode($reflector, $node, $locatedSource, $namespace, $positionInNode);
+        }
+
+        if ($node instanceof Node\Expr\FuncCall
+            && $node->name instanceof Node\Name
+            && $node->name->toLowerString() === 'define'
```

Comment:
Same as above: can't remember if the name is the FQN

Reply:
Changed the condition to `&& $node->name instanceof Node\Name\FullyQualified`

Reply:
Similar to https://github.com/Roave/BetterReflection/pull/471#discussion_r284239629 - need to still use `FullyQualified` and fallback if function lookup fails

Reply:
Would it be enough to solve it in one place? This class is internal.

Reply:
https://github.com/Roave/BetterReflection/pull/471/commits/68cd029675a3397660dd726960bdb5c71a7aec31

I used a hack to solve circular dependency. I would be very happy for better suggestion how to solve it here.

Reply:
Works for me for now :+1: